testing/xfpga/test_usrclk_c.cpp

```diff
@@ -80,8 +80,8 @@ class usrclk_c
     ASSERT_GT(num_matches_, 0);
 
     ASSERT_EQ(xfpga_fpgaGetProperties(nullptr, &filter_accel_), FPGA_OK);
-    ASSERT_EQ(fpgaPropertiesSetDeviceID(filter_accel_,
-                                        platform_.devices[0].device_id), FPGA_OK);
+    auto devid = platform_.devices[0].device_id + platform_.devices[0].num_vfs;
```

Comment:
Not convinced this will always be the case.

Reply:
You're right. This will work for now but fixing this the right way would require more work to the testing hw tables and data structures.